FAQ UseWasabi: add privacy progress (#1136)
zkSNACKs/WasabiDoc: docs/FAQ/FAQ-UseWasabi.md

When this is signed on the device with the private key (like an offline laptop running Electrum wallet or a hardware wallet), then you can broadcast the signed transaction using the `Broadcast Transaction` tab in the `Tools` menu.
:::

:::details
### What does the privacy progress mean?

> The privacy progress indicates the percentage of the wallet balance in coins above the anonscore target.

Reviewer: This is not correct. Even if all your coins did not reach your anonscore target you will get some privacy progress.

```suggestion
The privacy progress tile represents the percentage of how much private the wallet is.

It considers the anonymity score weighted amounts instead of just amounts that reached the anonymity score threshold.

The private value is the total amount that reached the anonscore threshold.
```

Reviewer 2: Minor edit to the line 195 suggestion by @yahiheb. 

```suggestion
The privacy progress tile represents the percentage of how private the wallet is.
```
or
```suggestion
The privacy progress tile represents the percentage of how much privacy the wallet has.
```

Reviewer: Thanks, I would go with the first one.

When this is signed on the device with the private key (like an offline laptop running Electrum wallet or a hardware wallet), then you can broadcast the signed transaction using the `Broadcast Transaction` tab in the `Tools` menu.
:::

:::details
### What does the privacy progress mean?

> The privacy progress tile represents the percentage of how much private the wallet is.

Reviewer: ```suggestion
The privacy progress tile represents the percentage of how private the wallet is.
```
